avr/cores/megacommand/CommonTools/helpers.c

```diff
@@ -26,6 +26,9 @@
  * @{
  **/
 
+const uint8_t _bvmasks[]  PROGMEM = {0b00000001, 0b00000010, 0b00000100, 0b00001000, 0b00010000, 0b00100000, 0b01000000, 0b10000000};
```

Comment:
Because we are optimizing for speed, I think these should be loaded in to SRAM.

"I think you will find it takes 3 clock cycles to read from PROGMEM compare to two for RAM"
https://forum.arduino.cc/index.php?topic=134782.0

Reply:
16bytes vs 1 cycle :thinking:

Reply:
we'd probably remove `PROGMEM` here. The code should not work because I was not using pgm_read_byte (the `LPM` instruction), but a simple `LD` instruction which only works for RAM.

Reply:
you're right, `LPM` is 3cyc.
https://www.microchip.com/webdoc/avrassembler/avrassembler.wb_LPM.html